presto-main/src/test/java/com/facebook/presto/operator/scalar/TestScalarValidation.java

```diff
@@ -370,13 +370,13 @@ public static long good2(
     @Test
     public void testValidTypeParametersForConstructors()
     {
-        extractScalars(ConstructorWithValidTypeParameters.class);
+        extractParametricScalar(ConstructorWithValidTypeParameters.class);
```

Comment:
I see. so `extractScalars` is for function set class and `extractParametricScalar` is for (parametric) function definition class.